adoc/Release-Notes.adoc

```diff
@@ -107,6 +107,8 @@ endif::[]
 [id='sec.1_5_1.issue']
 ==== Known Issues
 
+* Based on problems spotted in HA deployments and upgrades, it is recommended that you start any internal database role with a count of 1 first and then scale up to the needed number. This does not impact usage when connected to external databases.
```

Comment:
So far, we have noticed this problem only during HA deployments but better to provide instructions for starting with 1 PXC instance for both deployments and upgrades.

Reply:
@btat should we update the docs first to reflect this and have the revised release notes point there?

Reply:
We may have to be precise in what the problem is with what happens with multiple instances.

Reply:
The problem is captured, albeit very briefly, in the next paragraph (?):
"On occasion, the `uaa` pod may fail to start due to database migration failures." We should add to that : " `uaa` and possibly some other roles may fail to start due to database migration failures."

Reply:
> 
> 
> @btat should we update the docs first to reflect this and have the revised release notes point there?

For upgrades, should I add back the different upgrade scenarios similar to 1.5 (https://documentation.suse.com/suse-cap/1.5/single-html/cap-guides/#sec-cap-update)?

For deployments, the docs always had initial steps as SA to provide a kind of minimal deployment. In the separate chapter for HA, we listed a prerequisite of starting with SA mysql instances before transitioning to HA and that prereq wasn't removed for 1.5.1.

Reply:
@btat yes, let's add back the upgrade scenarios similar to 1.4.1 to 1.5.

Reply:
PR for docs made.

For the general upgrade section the link is https://documentation.suse.com/suse-cap/1.5.1/single-html/cap-guides/#sec-cap-update

Link (once published) for clusters currently using `config.HA` will be https://documentation.suse.com/suse-cap/1.5.1/single-html/cap-guides/#sec-cap-upgrade-config-ha

Link (once published) for clusters currently using custom sizing will be https://documentation.suse.com/suse-cap/1.5.1/single-html/cap-guides/#sec-cap-upgrade-custom-sizing

Link (once published) for clusters in SA will be https://documentation.suse.com/suse-cap/1.5.1/single-html/cap-guides/#sec-cap-upgrade-no-ha

Reply:
Thanks @btat. Once the links are live, I'll attempt an initial wording with links to those sections.